sources/packages/web/src/types/vuetify.ts

```diff
@@ -52,3 +52,5 @@ export interface SelectItemType {
   title: string;
   value: string | number;
 }
+
+export type Event = { target: { value: string } };
```

Comment:
I am not sure why was it created as a type. It seems that an interface would be more appropriate following the others in the same file.
`VForm` is the type to allow the concatenation only.
Not a blocker.